### DISCUSSION TITLE
Domain configuration postal prefix

### DISCUSSION THREAD

**Discussion by programmeddeath1** [postalserver/postal#2119] [Help with using Postal] [answered]
Hi team,

Just starting with my postal installation and wanted some help.

The documentation shows mx record addition as postal.example.com. I am assuming this is for the domain www.example.com.

Should all the records be prefixed with the "postal" value or that is simply to show sub-domain usability?

Basically should my records be postal.example.com or example.com in my DNS configuration?

Thanks in advance!

**Comment by willpower232** [comment, marked as answer]
Using postal as a subdomain is much easier so you know those DNS records are always Postal related.

Also if you have a separate DNS record for your server, CNAME'ing the apex of your domain (example.com) could lead you to have a confusing time with DNS as all the other records won't be as you expect.
